components/rockset/actions/add-documents/add-documents.mjs

```diff
@@ -0,0 +1,43 @@
+// legacy_hash_id: "a_bKiPAo"
+import { axios } from "@pipedream/platform";
+
+export default {
+  key: "rockset-add-documents",
+  name: "Add Documents",
+  description: "Add documents to a collection in Rockset. Learn more at https://docs.rockset.com/rest/#adddocuments.",
+  version: "0.1.1",
+  type: "action",
+  props: {
+    rockset: {
+      type: "app",
+      app: "rockset",
+    },
+    data: {
+      type: "any",
+      description: "Array of JSON documents. Learn more at https://docs.rockset.com/rest/#adddocuments.",
+    },
+    workspace: {
+      type: "string",
+      description: "Name of the workspace. ",
+    },
+    collection: {
+      type: "string",
+      description: "Name of the collection.",
+    },
+  },
+  async run({ $ }) {
+    const data = {
+      "data": this.data,
+    };
+
+    return await axios($, {
+      method: "POST",
+      url: `https://api.rs2.usw2.rockset.com/v1/orgs/self/ws/${this.workspace}/collections/${this.collection}/docs
```

Comment:
Is it expected that there's a newline here between the URL and the trailing backtick?

Reply:
It looks that newline comes from the legacy action. I think it can be removed.

Reply:
Removed the newline between the URL and trailing backtick